mkdocs/config/config_options.py

```diff
@@ -1141,7 +1141,7 @@ def load_plugin(self, name: str, config) -> plugins.BasePlugin:
         # Only if the plugin doesn't have its own "enabled" config, apply a generic one.
         if not any(pair[0] == 'enabled' for pair in plugin.config_scheme):
             if not config.get('enabled', True):
-                log.info(f"Plugin '{inst_name}' is disabled in the config, skipping.")
+                log.warning(f"Plugin '{inst_name}' is disabled in the config, skipping.")
```

Comment:
you meant `debug` right?

tbh I was fine with INFO but I guess DEBUG is more in line with how other plugins behave

Reply:
Oh what, yes I got things totally mixed up